mods/cnc/rules/structures.yaml

```diff
@@ -309,6 +336,18 @@ HAND:
 
 AFLD:
 	Inherits: ^BaseBuilding
+	HitShape:
```

Comment:
I just realized it might be worth it to account for higher-than-0 Z-offsets in the combat overlay, by drawing a line to the ground WPos, like we do for blocking height.

Anyway, I'll see what I can do about the airfield, but I'm not sure what changes you have in mind for the comm center (HQ or EYE, btw?). Target offsets are already footprint-based, and making the right half higher than the occupied cell is partially to cover the main buildings' sprite, partially to make the center target offset not sit at the edge/inner corner of the combined shape, otherwise attacks from north-east would have trouble dealing full damage if their spread is low and/or inaccuracy is high.